Libraries/Opc.Ua.Client/TraceableSession.cs

```diff
@@ -390,11 +390,11 @@ public ReferenceDescription FindDataDescription(NodeId encodingId)
         }
 
         /// <inheritdoc/>
-        public Task<DataDictionary> FindDataDictionary(NodeId descriptionId)
+        public async Task<DataDictionary> FindDataDictionary(NodeId descriptionId)
```

Comment:
Also, if the method returns a Task, it needs to have the suffix Async: FindDataDictionaryAsync